WordPress/Classes/Models/AtMentionSuggestion+CoreDataClass.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CoreData
+
+@objc(AtMentionSuggestion)
+public class AtMentionSuggestion: NSManagedObject {
```

Comment:
It took me a little bit to understand that `At` refers to @, I was thinking in ATomic :P

What do you think about rename this entity to represent what it is? Like `UserSuggestion`? Then in the future we will also have `SiteSuggestion`. I guess it makes a little bit easier to understand. Wdyt?

Reply:
Good idea. I spent too much time with @-mentions that I'd overlooked how AtMention might be confused with **At**omic.

Reply:
Renamed in af9fc2cd7f08defb3157de4eef8216df55d827c2